sanic/worker/loader.py

```diff
@@ -109,18 +110,28 @@ class CertLoader:
     _creator_class: Type[CertCreator]
 
     def __init__(self, ssl_data: Dict[str, Union[str, os.PathLike]]):
+        self._ssl_data = ssl_data
+
         creator_name = ssl_data.get("creator")
-        if creator_name not in ("mkcert", "trustme"):
-            raise RuntimeError(f"Unknown certificate creator: {creator_name}")
-        elif creator_name == "mkcert":
-            self._creator_class = MkcertCreator
-        elif creator_name == "trustme":
-            self._creator_class = TrustmeCreator
+        if not creator_name:
+            return
+        else:
+            if creator_name not in ("mkcert", "trustme"):
+                raise RuntimeError(
+                    f"Unknown certificate creator: {creator_name}"
+                )
+            elif creator_name == "mkcert":
+                self._creator_class = MkcertCreator
+            elif creator_name == "trustme":
+                self._creator_class = TrustmeCreator
 
         self._key = ssl_data["key"]
         self._cert = ssl_data["cert"]
         self._localhost = cast(str, ssl_data["localhost"])
 
     def load(self, app: SanicApp):
+        if not hasattr(self, "_creator_class"):
+            return process_to_context(self._ssl_data)
```

Comment:
I didn't look into the sensibility of this line. It is late and I have to sleep.